newIDE/app/src/EventsSheet/InstructionEditor/InstructionOrObjectSelector.js

```diff
@@ -356,18 +370,33 @@ export default class InstructionOrObjectSelector extends React.PureComponent<
                               : undefined,
                           })
                         )}
-                      {!isSearching &&
-                        currentTab === 'free-instructions' &&
-                        renderInstructionOrExpressionTree({
-                          instructionTreeNode: this.freeInstructionsInfoTree,
-                          onChoose: onChooseInstruction,
-                          iconSize,
-                          selectedValue: chosenInstructionType
-                            ? getInstructionListItemValue(chosenInstructionType)
-                            : undefined,
-                          initiallyOpenedPath: this.initialInstructionTypePath,
-                          selectedItemRef: this._selectedItem,
-                        })}
+                      {!isSearching && currentTab === 'free-instructions' && (
+                        <>
+                          {renderInstructionOrExpressionTree({
+                            instructionTreeNode: this.freeInstructionsInfoTree,
+                            onChoose: onChooseInstruction,
+                            iconSize,
+                            selectedValue: chosenInstructionType
+                              ? getInstructionListItemValue(
+                                  chosenInstructionType
+                                )
+                              : undefined,
+                            initiallyOpenedPath: this
+                              .initialInstructionTypePath,
+                            selectedItemRef: this._selectedItem,
+                          })}
+                          {onClickMore && (
+                            <Line justifyContent="center">
+                              <RaisedButton
+                                primary
+                                icon={<Add />}
+                                onClick={onClickMore}
+                                label={<Trans>Search new extensions</Trans>}
```

Comment:
You suggested  "Discover other actions/conditions in behaviors/extensions", maybe we could have here:

"Search for new actions in extensions" or "Search for new conditions in extensions" (according to isCondition)

or "Find new [...]" if "Search for" is too long?